src/parse_items.rs

```diff
@@ -0,0 +1,43 @@
+use nom::error::Error;
```

Comment:
```suggestion
// For the full copyright and license information, please view the LICENSE
// file that was distributed with this source code.

use nom::error::Error;
```

Same for the rest of the files

Reply:
Have added it to all files